.github/workflows/ci.yml

```diff
@@ -44,7 +44,4 @@ jobs:
         run: pnpm lint
 
       - name: Run tests 🧪
-        run: pnpm test
-
-      - name: Run coverage 📈
```

Comment:
Nope before this PR the tests were runned twice, without coverage first and then with coverage, now we only run the tests once but without coverage because currently this repo doesn't have any coverage requirements rules setup.